Tools/Scripts/webkitpy/benchmark_runner/run_benchmark.py

```diff
@@ -68,6 +68,8 @@ def config_argument_parser():
 
     parser.add_argument('browser_args', nargs='*', help='Additional arguments to pass to the browser process. These are positional arguments and must follow all other arguments. If the pass through arguments begin with a dash, use `--` before the argument list begins.')
 
+    parser.add_argument('--http-server-type', default="twisted", choices=["twisted", "base"], help="Specify the http server to use for the webserver benchmark runner.")
```

Comment:
`base` does not seem to be accurate enough, how about `builtin` or `py3-builtin`?

Reply:
I'll rename it to `builtin`